crates/ruff_python_formatter/src/trivia.rs

```diff
@@ -193,9 +193,18 @@ pub(crate) enum TokenKind {
     /// `in`
     In,
 
+    /// `as`
+    As,
```

Comment:
@davidszotten thanks for implementing the keyword lexing in a generic way. It was a piece of cake to add the new keywords that I needed.